app/code/Magento/Fedex/Model/Carrier.php

```diff
@@ -1571,7 +1571,7 @@ private function processTrackingDetails(\stdClass $trackInfo)
         ];
 
         if (!empty($trackInfo->ShipTimestamp)) {
-            $datetime = \DateTime::createFromFormat(\DateTime::ISO8601, $trackInfo->ShipTimestamp);
+            $datetime = \DateTime::createFromFormat('U', strtotime($trackInfo->ShipTimestamp));
```

Comment:
According to FedEx API docs, date time should be with time zone, but sometimes API returns it without timezone. This fix is incorrect.